resources/views/footer.blade.php

```diff
@@ -1,24 +1,100 @@
+    'reservedRights' => true, // Show "All rights reserved"...
+    'policy' => [], // Needs `url` and `label` to display the Privacy Policy link...
+    'terms' => [], // Needs `url` and `label` to display the Terms of Service link...
+    'socials' => [
+        [
+            'icon' => 'brands.solid.twitter',
+            'url' => trans('ui::urls.twitter')
+        ],
+        [
+            'icon' => 'brands.solid.linkedin',
+            'url' => trans('ui::urls.linkedin')
+        ],
+        [
+            'icon' => 'brands.solid.facebook',
+            'url' => trans('ui::urls.facebook')
+        ],
+        [
+            'icon' => 'brands.solid.youtube',
+            'url' => trans('ui::urls.youtube')
+        ],
+        [
+            'icon' => 'brands.solid.github',
+            'url' => trans('ui::urls.github')
+        ],
+        [
+            'icon' => 'brands.solid.telegram',
+            'url' => trans('ui::urls.telegram')
+        ],
+    ],
 ])
 
-<div {{ $attributes->merge(['class' => 'border-t bg-theme-secondary-900 border-theme-secondary-800']) }}>
-    <div class="{{ $wrapperClass }} flex-col mx-auto">
-        <div class="flex flex-col items-start lg:flex-row lg:items-center justify-between space-y-3 lg:space-y-0 py-5 @if ($border) border-t border-theme-secondary-800 @endunless">
-            <x-ark-footer-copyright
-                :is-ark-product="$isArkProduct"
-                :copy-text="$copyText"
-                :class="$copyClass"
-                :copyright-slot="$copyrightSlot"
-            />
-
-            <x-ark-footer-social :networks="$socials" />
+<footer {{ $attributes->class('border-t border-theme-secondary-800')->class($backgroundColor) }}>
+    <div class="{{ $wrapperClass }} mx-auto justify-between items-center py-5 space-y-4 lg:flex lg:space-y-0">
+        <div class="text-sm font-semibold leading-6 break-words">
+            <span>
+                {{ date('Y') }}
+                &copy;
+                @if ($name && ! $url)
+                    {{ $name }}
+                @elseif ($name && $url)
+                    <a href="{{ $url }}" class="transition-default hover:text-theme-secondary-200">{{ $name }}</a>
+                @elseif (isset($creator['label']) || is_string($creator))
+                    <span>
+                        @lang ('ui::pages.footer.made_with_love') <a href="{{ $creator['url'] ?? url('/') }}" class="{{ $linkClass }}" target="_blank" rel="noopener">{{ is_string($creator) ? $creator : $creator['label'] }}</a>
+                    </span>
+                @endif
+            </span>
+
+            @if ((is_string($creator) || isset($creator['label'])) && $name)
+                <span class="mx-0.5">|</span>
+                <span>
+                    @lang ('ui::pages.footer.made_with_love') <a href="{{ $creator['url'] ?? url('/') }}" class="{{ $linkClass }}" target="_blank" rel="noopener">{{ is_string($creator) ? $creator : $creator['label'] }}</a>
```

Comment:
this has `target="_blank"` while we open urls in the same window when they point to the existing domain (e.g. ark.io links to ark.io as "made with love by" thing and does not open in a new tab when clicked, similar for ardenthq). 

basically it should depend on the URL whether it uses `_blank` or not